src/main/xerus/monstercat/api/APIConnection.kt

```diff
@@ -74,23 +74,50 @@ class APIConnection(vararg path: String) : HTTPQuery<APIConnection>() {
 	fun getTracks() =
 		parseJSON(TrackResponse::class.java)?.results
 	
+	fun postLogin(username: String, password: String) =
+		post(HttpPost(uri).apply {
+			setHeader("Accept", "application/json")
+			setHeader("Content-type", "application/json")
+			entity = StringEntity("{\"email\":\"$username\",\"password\":\"$password\"}")
```

Comment:
```suggestion
			entity = StringEntity("""{"email":"$username","password":"$password"}""")
```

Reply:
Wouldn't this not work because $variable doesn't work in plain strings ?

Reply:
Nevermind just checkeed the doc it does work.